api/controllers/game/stalemate.js

```diff
@@ -0,0 +1,56 @@
+module.exports = async function (req, res) {
+  try {
+    const game = await Game.findOne({ id: req.session.game }).populate('players');
```

Comment:
I've been working on doing a bit more destructuring, is that something you'd be in interested in using across the codebase?
```suggestion
  try {
    const { game, pNum } = req.session; 
    const game = await Game.findOne({ id: req.session.game }).populate('players');
```

Reply:
Yes I think destructuring is nice. Will use (though in this case it can't be the exact suggestion due to name collision, and we'll want to use the variable in the method call